Header header
sensor_msgs/Image hand_left
sensor_msgs/Image hand_right
std_msgs/Int8[] left_bounding_box
std_msgs/Int8[] right_boundind_box 